.github/workflows/meta-tuist.yml

```diff
@@ -73,7 +74,10 @@ jobs:
           bundle install --jobs 4 --retry 3
       - name: Build
         run: |
-          ./fourier generate
+          swift build
+          swift run tuist fetch
+          swift run tuist cache warm
+          ./fourier generate tuist
```

Comment:
What about changing Fourier generate to do the fetch and cache?

Reply:
https://github.com/tuist/tuist/pull/4280/commits/c7873d173391c0e4844a7b6ac87b06e87c4fa2b7

I changed to `cache warm` all targets when running `./fourier generate` - I think that's fair as most of them should usually be now from remote cache. Let me know what you think @danyf90. I can also allow developers to specify `./fourier generate --warm-dependencies-only` to skip warming internal targets.

Reply:
Actually, I gave it another thought and I don't think this is really developer-friendly. Those additional steps are needed for CI only whereas `fourier` is meant for contributors.